src/lib/autoinstall/autosetup_helpers.rb

```diff
@@ -229,6 +232,16 @@ def autosetup_firewall
       Yast::Profile.remove_sections("firewall") if !need_second_stage_run?
     end
 
+    # Check security policies
+    #
+    # If any of the rules of the enabled policies fails, it enables the confirm mode so the user has
+    # a chance to review and fix the potential issues.
+    def autosetup_security_policies
+      target_config = Y2Security::SecurityPolicies::TargetConfig.new
+      rules = Y2Security::SecurityPolicies::Manager.instance.failing_rules(target_config)
```

Comment:
I guess that something in here makes the failing rules visible to the user but I only know because of the PR screenshot. Please add a comment so that a casual reader of the code will realize it too.

Reply:
It just enables the confirmation mode, so you get the confirmation screen later. This behavior is described in the PR (but perhaps it needs better wording).